CAndC++.md

```diff
@@ -55,6 +55,8 @@ greater performance will be added soon after, including:
   [LLVM's vector extensions]: http://clang.llvm.org/docs/LanguageExtensions.html#vectors-and-extended-vectors
   [GCC's vector extensions]: https://gcc.gnu.org/onlinedocs/gcc/Vector-Extensions.html
 
+Note that inline assembly or other platform-specific features are not supported.
```

Comment:
This is a little confusing, because WebAssembly C/C++ compilers do support some forms of inline assembly. The mental model is that it *is* platform-specific, because the platform is WebAssembly itself.